docs/mobile-apps/features/network-capture.md

```diff
@@ -0,0 +1,104 @@
+---
+id: network-capture
+title: Network Traffic Capture
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Network Traffic Capture is a functionality that gives you the ability to record network traffic and HTTP/HTTPS request headers during your native Android and iOS/iPadOS mobile app app tests. This provides granular insight into your app’s behavior, facilitates debugging, and helps you determine which development team is responsible for application failure.
```

Comment:
We only able to capture HTTP/HTTPS network traffice. So something like:
"that gives you the ability to record network traffic for HTTP/HTTPS requests"